src/components/form/FormFieldOptionGroup.vue

```diff
@@ -0,0 +1,183 @@
+<script lang="ts">
+/**
+ * FormFieldOptionGroup Component
+ *
+ * The `FormFieldOptionGroup` displays option group input.
+ *
+ * @description * Use this component to render option groups with icon title
+ * and description in the form of cards.
+ *
+ * Note: This component is commonly used in `FormRegisterChallenge`.
+ *
+ * @props
+ * - `modelValue` (string, required): The object representing user input.
+ * - `name` (string, required): The name used for id and test selectors.
+ * - `label` (string, required): The translation key for the label.
+ *
+ * @events
+ * - `update:modelValue`: Emitted as a part of v-model structure.
+ *
+ * @example
+ * <form-field-option-group v-model="value" />
+ *
+ * @see [Figma Design](https://www.figma.com/file/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?type=design&node-id=5350%3A24169&mode=dev)
+ */
+
+// libraries
+import { colors } from 'quasar';
+import { defineComponent, computed } from 'vue';
+
+// composables
+import { useValidation } from 'src/composables/useValidation';
+
+// types
+import { ConfigGlobal } from '../types';
+
+// config
+const rideToWorkByBikeConfig: ConfigGlobal = JSON.parse(
+  process.env.RIDE_TO_WORK_BY_BIKE_CONFIG,
+);
+
+export default defineComponent({
+  name: 'FormFieldOptionGroup',
+  props: {
+    modelValue: {
+      type: String,
+      required: true,
+    },
+    name: {
+      type: String,
+      required: true,
+    },
+    label: {
+      type: String,
+      required: true,
+    },
+  },
+  emits: ['update:modelValue'],
+  setup(props, { emit }) {
+    const inputValue = computed({
+      get() {
+        return props.modelValue;
+      },
+      set(value: string) {
+        emit('update:modelValue', value);
+      },
+    });
+
+    const { isFilled } = useValidation();
+
+    const options = [
```

Comment:
Missing type?

Reply:
Fixed, making use of the updated `FormOption` type.